proto/api/v2/compliance_integration_service.proto

```diff
@@ -17,6 +17,7 @@ message ComplianceIntegration {
   string namespace = 5;
   // Collection of errors that occurred while trying to obtain compliance operator health info.
   repeated string status_errors = 6;
+  bool read_only = 7;
```

Comment:
This actually resolves what I intended for https://issues.redhat.com/browse/ROX-22167.  So 2 birds 1 stone.

Reply:
Great, closed the other ticket as a won't do

Reply:
Oh, it is already closed, nothing to do here :sleeping_bed: